cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseVersionSpecificFactory.java

```diff
@@ -53,6 +53,9 @@ public T get() {
         case HBASE_10_CDH56:
           instance = createInstance(getHBase10CHD550ClassName());
           break;
+        case HBASE_12_CDH57:
```

Comment:
Not completely your PR but can you please arrange them in order of hbase version